app/components/modals/event-delete-modal.js

```diff
@@ -0,0 +1,20 @@
+import Ember from 'ember';
+import ModalBase from 'open-event-frontend/components/modals/modal-base';
+import FormMixin from 'open-event-frontend/mixins/form';
+import { lowerCase } from 'lodash';
+
+const { computed } = Ember;
+
+export default ModalBase.extend(FormMixin, {
+  isSmall: true,
+
+  actions: {
+    deleteEvent() {
```

Comment:
Bubble this action up to the controller. To be handled by the controller. 

Lets follow data-down, action-up approach